### PR TITLE
Fix #3246 - document Stable sorts

### LINKED ISSUE #3246: Document -Stable parameter for Sort-Object
Issue Details
-----------------

A -Stable parameter has been added to Sort-Object in [PR7862](https://github.com/PowerShell/PowerShell/pull/7862) that parameter needs to be documented.

Version(s) of document impacted
------------------------------
- [X] Impacts 6.next document
- [ ] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [X] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

### PR DESCRIPTION
**Pull request by sdwheeler**
Update Sort-Object docs for new -Stable parameter.

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[6173623](https://github.com/sdwheeler/PowerShell-Docs/pull/4141/commits/617362322bdc7896a8367b83f8b21a4c5b22cf9b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Utility/Sort-Object.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3246/reference/6/Microsoft.PowerShell.Utility/Sort-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/sort-object?view=powershell-6&branch=pr-en-us-4141) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C9%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904091640343849-4141%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=MlA94X%2F6nX0LS0ZjcYapi%2BVDj4zJCLFmYUV0nGbDNxU%3D&st=2019-04-09T16%3A39%3A44Z&se=2019-05-10T16%3A44%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[93d9580](https://github.com/sdwheeler/PowerShell-Docs/pull/4141/commits/93d95805df34d758a571cdf72cede815901c17d8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Utility/Sort-Object.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3246/reference/6/Microsoft.PowerShell.Utility/Sort-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/sort-object?view=powershell-6&branch=pr-en-us-4141) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C9%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904091643048402-4141%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=WEgX%2B6WXOec754d9LKf3uRhMg4mGBc4E8QNByCsXuFw%3D&st=2019-04-09T16%3A44%3A16Z&se=2019-05-10T16%3A49%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Utility/Sort-Object.md [hunk 1]
```diff
@@ -313,11 +319,69 @@ the strings to integers. In the sample code, `[int]` converts the string to an i
...
+19
+11
+8
+14
+5
+17
+2
+20
+
++PS> 1..20 |Sort-Object {$_ % 3} -Stable
```
Comment (DCtheGeek) [inline reference/6/Microsoft.PowerShell.Utility/Sort-Object.md]: There's a `+` in front of the row which will break the prompt.

Comment (sdwheeler) [reply reference/6/Microsoft.PowerShell.Utility/Sort-Object.md]: Fixed. It was a copy/paste from my prompt.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[65d15cc](https://github.com/sdwheeler/PowerShell-Docs/pull/4141/commits/65d15cc7b7dcd61ce223f665499a339f8fd07249)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Utility/Sort-Object.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3246/reference/6/Microsoft.PowerShell.Utility/Sort-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/sort-object?view=powershell-6&branch=pr-en-us-4141) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C9%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904091810159632-4141%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4s9ioX2nPBjwseq484O%2BxwLNA7shfaLQQwi00v2Xz4c%3D&st=2019-04-09T18%3A09%3A52Z&se=2019-05-10T18%3A14%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**